### ISSUE TITLE
You are able to fix Brain and ears through eye surgery

### ISSUE THREAD

**Issue by Joonasm** [ParadiseSS13/Paradise#11194]
**Problem Description**:
When doing eye surgery and healing organs you heal all the organs in 

**What did you expect to happen**: 
To only repair the eyes

**What happened instead**:
All the organs were repaired

**Why is this bad/What are the consequences:**
It allows brain damage to be fixed much faster through surgery as you dont have to cut through the skull

**Steps to reproduce the problem**:
Do organ manipulation surgery on eyes. Heal the organs

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was anything else out of the ordinary happening?)**:
I was playing a medical borg when i found out
Anything else you can tell us.

**Comment by SkeletalElite** [comment]
I don't think this should be changed really. Just make's fixing brain damage through surgery even worse if you do and mannitol is already super easy to make and available in the vendors so I think it's better that this stays as a trick to do surgery brain fixing faster since there's already very little reason to do it over using mannitol.

**Comment by Mitchs98** [comment]
I agree. It should stay as is until non linear surgery is ported(if ever). It's an amazing timesaver.
